packages/icons/src/index.js

```diff
@@ -276,4 +287,15 @@ export {
   Wallet,
   Whatsapp,
   Youtube,
+  FlagArgentina,
+  FlagBrazil,
+  FlagChile,
+  FlagGermany,
+  FlagIreland,
+  FlagItaly,
+  FlagMexico,
+  FlagPortugal,
+  FlagSpain,
+  FlagUK,
+  FlagUS,
```

Comment:
What do you think about having a object inside the icons named as `Flags` to improve readability and thinking about some clients usages that will probably need to import all "flag" icons to create a dictionary for example, @mmartinsoliv? And maybe remove the repeated word Flag, but I'm not so sure if those approaches are the best thinking on the API expectations

Reply:
I thought it was a good suggestion! But I believe we should keep this decision about importing the icons on the project side. If we were to analyze some APIs about icon libs, it ends up not being a standard to export in this way.

And anyway we can import like this:

```
import * as Icons from '@gympass/yoga-icons'

<Icons.FlagBrazil />
```

Reply:
Understood